### ISSUE TITLE
Negative Config Values Prevent Parsing

### ISSUE THREAD

**Issue by chrismurf** [aphysci/gravity#119]
Currently, it seems like negative values cannot be parsed.  Having the following line in a config file:
```ini
OriginLongitude = -73.208983
```
Will cause the following error to be emitted:
```bash
yacc: error line 8: syntax error, unexpected '-'
```
This can be worked around by using the following line and relying on the built-in math capabilities (yuck!).
```ini
OriginLongitude = 0 - 73.208983
```

**Comment by mbarger99** [comment]
An alternate workaround is to put the number if quotes:

OriginLongitude = "-73.208983"

It will be converted to a float when it's retrieved via getFloatParam.  But agreed, unary minus should work without quotes or a preceding 0.

**Comment by JaredDiCioccio** [comment]
Fixed in #145
